### DISCUSSION TITLE
Self-Hosted Supabase - Studio - Numerous 404s in Infra Logs and Network Tab

### DISCUSSION THREAD

**Discussion by adao** [supabase/supabase#28360] [Questions]
I just stood up the self-hosted docker version. I'm exploring the Studio dashboard and went to the logs section. There are a bunch of 404s in the network tab and in the logs.

<img width="827" alt="Screenshot 2024-08-04 at 3 50 01 AM" src="https://github.com/user-attachments/assets/73916d5d-7c41-44a9-b9e8-964f24710049">

<img width="725" alt="Screenshot 2024-08-04 at 3 50 47 AM" src="https://github.com/user-attachments/assets/84d84274-c13c-4b44-a270-1119d7461dfb">

There many requests to an /api route, which I cannot find the corresponding microservice for within kong.yml, so I think it resolves to Studio which is just the base route.

Then there are a few requests for web resources like js.map files also returning 404.

Not sure how to go about fixing these. Is this just the current state of things?

**Comment by haleksandre** [comment]
Have you figured it out? I too have multiple 404s in the logs pages. 

It seems like Studio is trying to load non-existing pages/endpoints https://github.com/supabase/supabase/tree/7ad4ad6ee5aad55a0e8833da77995f6560d228ac/apps/studio/pages/api/platform/projects/%5Bref%5D
 
Not sure if things were moved or deleted and the app wasn't updated accordingly & is pointing to empty internal API endpoints?
